crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1120,11 +1142,11 @@ bool LVFreeTypeFace::getGlyphInfo(lUInt32 code, LVFont::glyph_info_t *glyph, lCh
     glyph->blackBoxY = (lUInt16)( FONT_METRIC_TO_PX( _slot->metrics.height ) );
     glyph->originX =   (lInt16)( FONT_METRIC_TO_PX( _slot->metrics.horiBearingX ) );
     glyph->originY =   (lInt16)( FONT_METRIC_TO_PX( _slot->metrics.horiBearingY ) );
-    glyph->width =     (lUInt16)( FONT_METRIC_TO_PX( myabs(_slot->metrics.horiAdvance )) );
+    glyph->width =     (lUInt16)( FONT_METRIC_TO_PX( myabs(_slot->metrics.horiAdvance + _synth_weight_advance_add )) );
     if (glyph->blackBoxX == 0) // If a glyph has no blackbox (a spacing
         glyph->rsb =   0;      // character), there is no bearing
     else
-        glyph->rsb =   (lInt16)(FONT_METRIC_TO_PX( (myabs(_slot->metrics.horiAdvance)
+        glyph->rsb =   (lInt16)(FONT_METRIC_TO_PX( (myabs(_slot->metrics.horiAdvance + _synth_weight_advance_add)
```

Comment:
I have doubts about this synth_weight_advance_add as mentionned above - but this feels a bit quicky'n'hacky putting it all on the right side bearing.

Reply:
To be honest, I have not studied what it is at all, added by analogy. Think you need to remove it?